LayoutTests/inspector/unit-tests/steps-timing-function.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../http/tests/inspector/resources/inspector-test.js"></script>
+<script>
+function test() {
+    let suite = InspectorTest.createSyncSuite("WI.StepsTimingFunction");
+
+    suite.addTestCase({
+        name: "WI.StepsTimingFunction.fromString.Valid",
+        test() {
+            let test = (inputString, expectedString) => {
+                expectedString ||= inputString;
+                InspectorTest.expectEqual(WI.StepsTimingFunction.fromString(inputString).toString(), expectedString, `Should parse '${inputString}' as '${expectedString}'.`);
+            };
+
+            test("steps(1)", "step-end");
+            test("steps(2, jump-start)");
+            test("steps(3, jump-end)");
+            test("steps(4, jump-none)");
+            test("steps(5, jump-both)");
+            test("steps(6, end)");
+            test("steps(7, start)");
+            test("step-start");
+            test("step-end");
```

Comment:
NIT / personal preference: it was slightly ambiguous which `test()` this refers to because there are multiple declarations of that function name in this file. Nothing bad, but it required me slow down to figure out the scope in my head first. It helped that these were called `testPass()`and `testFail()` in the original file.